packages/blade/package.json

```diff
@@ -114,6 +114,7 @@
     "postinstall": "patch-package"
   },
   "dependencies": {
+    "scrollyfills": "1.0.0",
```

Comment:
needed in dependencies? :( 

Whats the size?

Reply:
Was debugging and see if this lib works, it didn't. We can remove it.